com.github.dynobo.sphereview.yml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.dynobo.sphereview
+command: sphereview
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.node24
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/node24/bin
+  build-args:
+    - --share=network
+  env:
+    CARGO_HOME: /run/build/rust-flatpak/cargo
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=xdg-pictures
+cleanup:
+  - /sbin
+  - /include
+  - /share/doc
+  - /share/pkgconfig
+  - /share/man
+
+modules:
+  - name: blueprint-compiler
+    buildsystem: meson
+    cleanup:
+      - "*"
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/blueprint-compiler
+        tag: v0.16.0
+
+  - name: sphereview
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        CARGO_HOME: /run/build/sphereview/cargo
+        CARGO_NET_OFFLINE: 'true'
+    build-commands:
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo build --offline --release --all-features
+      - install -Dm0755 target/release/sphereview ${FLATPAK_DEST}/bin/sphereview
+      - install -Dm644 com.github.dynobo.sphereview.svg
+        ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/com.github.dynobo.sphereview.svg
+      - install -Dm644 com.github.dynobo.sphereview.desktop
+        ${FLATPAK_DEST}/share/applications/com.github.dynobo.sphereview.desktop
+      - install -Dm644 com.github.dynobo.sphereview.appdata.xml
+        ${FLATPAK_DEST}/share/appdata/com.github.dynobo.sphereview.appdata.xml
+    sources:
+      - type: git
+        url: https://github.com/dynobo/sphereview.git
+        tag: v0.1.0
+        commit: f52d5df398b8bb6b1cdc550da2c0bcd95f8ec0ed
+      - type: file
+        path: com.github.dynobo.sphereview.svg
+      - type: file
+        path: com.github.dynobo.sphereview.desktop
+      - type: file
+        path: com.github.dynobo.sphereview.appdata.xml
+      - cargo-sources.json
+      - node-sources.json
```

Comment:
where are the node sources being used? you don't have npm etc. in your path and i don't see any build commands for that.

Reply:
> you don't have npm etc. in your path

Actually, I do, see [L10](https://github.com/dynobo/flathub/blob/bb031744a0faf5504f28e5e4df45ead2db5b77f1/com.github.dynobo.sphereview.yml#L10)

But I guess both L10 should be merged into L41... I'll do that.

> i don't see any build commands for that.

The npm  build is integrated in the rust build (because it depends on it). See [build.rs](https://github.com/dynobo/sphereview/blob/f52d5df398b8bb6b1cdc550da2c0bcd95f8ec0ed/build.rs#L32-L45)